pkg/test/usersignup/usersignup.go

```diff
@@ -1,13 +1,14 @@
 package usersignup
 
 import (
-	"crypto/md5" // nolint:gosec
-	"encoding/hex"
+	// nolint:gosec
```

Comment:
the `nolint` comment may not be needed anymore, since the import was moved.

Reply:
yes, I noticed it too, and fixed it in https://github.com/codeready-toolchain/toolchain-common/pull/269